src/plugin/settings/engine.cpp

```diff
@@ -480,9 +489,7 @@ nx::sdk::Error Engine::spawnMount()
 #endif
     if (mountRet.errCode != 0)
     {
-        std::string errorMessage = "Unable to launch mount with error: " + mountRet.output;
```

Comment:
Is there a reason to not log these messages using NX_PRINT?

Reply:
It turns out they're already logged after being returned. I only realized this when the mount actually failed recently.